Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM node:16-bullseye as builder
+RUN apt-get update && apt-get install -y cmake git build-essential python3-venv wget && rm -rf /var/lib/apt/lists/*
+
+# Install golang 
+RUN rm -rf /go && wget https://go.dev/dl/go1.19.1.linux-amd64.tar.gz -qO- | tar -C / -xz

```

Comment:
I think we're still on go1.18, and if using go1.19 imposes a dependency on that with the artifacts, it may be problematic.